mne_icalabel/features/topomap.py

```diff
@@ -17,69 +19,87 @@ def get_topomaps(
     ica: ICA,
     picks=None,
     res: int = 64,
-    outlines: Optional[str] = "head",
     image_interp: str = _INTERPOLATION_DEFAULT,  # 'cubic'
     border: Union[float, str] = _BORDER_DEFAULT,  # 'mean'
-    extrapolate: str = _EXTRAPOLATE_DEFAULT,  # 'head'
-):
+    extrapolate: str = _EXTRAPOLATE_DEFAULT,  # 'auto' -> 'head' for EEG, 'local' for MEG
+) -> Dict[str, NDArray[float]]:
```

Comment:
Not sure why but this breaks numpydoc validation.

Reply:
Yes.. I have no idea either. Looking into it.

Reply:
Now it's showing also:

```
WARNING: [numpydoc] Validation warnings while processing docstring for 'mne_icalabel.features.get_topomaps':
  RT03: Return value has no description

WARNING: [numpydoc] Validation warnings while processing docstring for 'mne_icalabel.features.get_topomaps':
  RT03: Return value has no description
```

Which is (probably?) related.

Reply:
Solved, It was coming from `sphinx_autodoc_typehints` which was not playing nicely with `numpydoc`.
I simply removed it, type-hints on the rendered website, even only in the signature, can become very messy.